admin/installer/setup_client.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Setup up a client compute instance to deploy sample workloads on an install
+# cluster using docker-compose.
+
+set -ex
+
+DOCKER_CERT_HOME="/root/.docker"
+UBUNTU_HOME="/home/ubuntu"
+
+# Get Postgres image.
+apt-get update
+sudo apt-get install -y postgresql-client
+
+# Get docker-compose.
+curl -L https://github.com/docker/compose/releases/download/1.5.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+chmod +x /usr/local/bin/docker-compose
+
+mkdir -p /home/ubuntu/postgres
+curl https://raw.githubusercontent.com/ClusterHQ/flocker/flocker-cloudformation-FLOC-3709/admin/installer/postgres/docker-compose-node0.yml > /home/ubuntu/postgres/docker-compose-node0.yml
```

Comment:
I think we should probably get rid of these.
The current documentation asks the user to download compose templates from the docs so there's no point trying to make them available this way.
I'll have to update the tests though.